src/badge/badge.module.css

```diff
@@ -16,14 +18,15 @@
 
 .badge {
     font-family: var(--reactist-font-family);
-    font-weight: var(--reactist-font-weight-medium);
+    font-weight: var(--reactist-font-weight-strong);
     font-size: var(--reactist-badge-font-size);
     text-transform: uppercase;
     letter-spacing: 0.1em;
     border-radius: 3px;
-    padding: 1px var(--reactist-spacing-xsmall);
+    padding: var(--reactist-badge-paddingY) var(--reactist-badge-paddingX);
     color: var(--reactist-badge-tint);
     background-color: var(--reactist-badge-fill);
+    line-height: calc(var(--reactist-badge-font-size) + 2 * var(--reactist-badge-paddingY));
```

Comment:
I think it would be a bit simpler if instead of applying a vertical padding, we just apply a line height and that's it. That alone would enforce the height and the vertical spacing around the text, even if it's not padding, right?

With that in mind, I would also hard-code the line-height to be 12px, instead of doing all this calculation. Something along these lines:

```css
.badge {
  padding: 0 var(--reactist-spacing-xsmall);
  line-height: calc(var(--reactist-badge-font-size) + 2);
}
```

I also feel that offering badge-specific CSS variables for the padding of the badge may be unnecessary. I'd rather add that API later if needed, than committing to it today. Any new CSS variables we add becomes public Reactist API that we later need to support or deprecate if we want to remove it.

Reply:
Err actually it seems like I still need to set the padding value to what I actually want the padding to be.

<img width="424" alt="Screenshot 2023-02-17 at 14 19 00" src="https://user-images.githubusercontent.com/1509326/219663338-75ce38ba-640d-4d49-a150-91f34577c0e9.png">

vs.

<img width="418" alt="Screenshot 2023-02-17 at 14 18 50" src="https://user-images.githubusercontent.com/1509326/219663334-aff85682-bc93-4ff2-82ff-662c923b33d3.png">